src/components/map/draw-tools/DrawingTools.tsx

```diff
@@ -4,16 +4,18 @@ import { Feature, Map } from 'ol';
 import { Geometry } from 'ol/geom';
 import VectorLayer from 'ol/layer/Vector';
 import VectorSource from 'ol/source/Vector';
-import { MdOutlineRectangle } from 'react-icons/md';
+import { PiPencilSimpleLineFill } from 'react-icons/pi';
 
-import './styles.scss';
+import { useMap } from '@/hooks/useMap';
 
-type DrawingToolsProps = {
-  map: Map;
-  drawRectangle: (drawingSource: VectorSource<Feature<Geometry>>) => void;
-};
+import { DrawingToolbox } from './DrawingToolbox';
+
+import './DrawingTools.scss';
 
-export const DrawingTools = ({ map, drawRectangle }: DrawingToolsProps) => {
+export const DrawingTools = () => {
+  const { map } = useMap();
+
+  const [isToolboxVisible, setIsToolboxVisible] = useState<boolean>(false);
```

Comment:
As we refer to the draggable element with the results in as `Toolbox`, this confused me at first.

Reply:
I'm happy to change this, maybe `isDrawingToolbox` instead?

Reply:
Yep that works, alternatively `isDrawingToolboxVisible`.